{{cookiecutter.github_repository}}/provisioner/roles/common/defaults/main.yml

```diff
@@ -28,6 +28,11 @@ base_ubuntu:
       - postgresql-client
       - libpq-dev
       - postgis
+      - ca-certificates
```

Comment:
Let move this to nginx role and only install if use_letsencrypt is true. @jainmickey Thoughts?